src/Ast/AST.php

```diff
@@ -461,6 +461,33 @@ public function toCode(): string
         };
     }
 
+    /**
+     * Create a ternary expression.
```

Comment:
Well, creates a conditional ?: expression.
You know my feelings about using the arity of an expression as its name ;)

Reply:
Comment changed :)